model/messaging/spans.yaml

```diff
@@ -118,6 +118,22 @@ groups:
           Message [correlation Id](https://www.rabbitmq.com/tutorials/tutorial-six-java#correlation-id) property.
       - ref: messaging.message.body.size
         requirement_level: opt_in
+      - ref: messaging.destination.name
+        note: |
+          In RabbitMQ, the destination is defined by an *exchange*, a *routing key* and for consumers, a *queue*.
+
+          `messaging.destination.name` SHOULD be set to:
+
+          - **On the producer side**: `{exchange}:{routing key}` when both values are present and non-empty.
+          When only one is available, only that value SHOULD be used. E.g., `{exchange}` or `{routing key}`.
+          Otherwise: `amq.default` when the default exchange is used and no routing key is provided
```

Comment:
what does the `amq` part signify? thanks

Reply:
It's how RabbitMQ "calls" the default (empty) exchange name:

> For convenience RabbitMQ maps AMQP 0-9-1's default exchange's blank name to 'amq.default' when performing permission checks.

https://www.rabbitmq.com/docs/access-control#authorisation
https://www.rabbitmq.com/tutorials/amqp-concepts#exchange-default
https://github.com/search?q=repo%3Arabbitmq%2Frabbitmq-server%20amq.default&type=code